### DISCUSSION TITLE
Allow RGB channels to be specified in viewer URL

### DISCUSSION THREAD

**Discussion by rsignell-usgs** [developmentseed/titiler#526] [General]
A nice enhancement for the viewer would be to allow specification of channels for RGB in the URL, like:
https://titiler.xyz/cog/viewer?url=https://prod-is-usgs-sb-prod-publish.s3.amazonaws.com/5b2859cce4b0592076260e4b/2017042FA_Chimney&R=1&G=2&B=3

That would allow sharing the URL so that it views properly, instead of telling folks they need to use the viewer GUI and to change the B channel from 4 to a 3!

**Comment by vincentsarago** [comment]
@rsignell-usgs with titiler 0.8.0 you can do this with the `/map` endpoint 

```
# bidx=4&bidx=1&bidx=2
https://titiler.xyz/cog/map?url=s3://njogis-imagery/2020/cog/L9A9.tif&bidx=4&bidx=1&bidx=2

# expression=(b4-b1)/(b4+b1)&rescale=-1,1&colormap_name=viridis
https://titiler.xyz/cog/map?url=s3://njogis-imagery/2020/cog/L9A9.tif&expression=%28b4-b1%29%2F%28b4%2Bb1%29&rescale=-1,1&colormap_name=viridis
```
